pages/windows/gpupdate.md

```diff
@@ -0,0 +1,20 @@
+# gpupdate
+
+> A tool to check and apply Windows Group Policy settings.
+> More information: <https://docs.microsoft.com/windows-server/administration/windows-commands/gpupdate>.
+
+- Check and apply updated Group Policy settings:
+
+`gpupdate`
+
+- Specify the target Group Policy settings to check for update:
+
+`gpupdate /Target=:{{Computer|User}}`
```

Comment:
`target` is in lower case in the docs, also we use `snake_case` for variables like `user`:
```suggestion
`gpupdate /target=:{{computer|user}}`
```